lib/tasks/guest_cleanup.rake

```diff
@@ -0,0 +1,47 @@
+namespace :fromthepage do
```

Comment:
Can we make this take a variable amount of time we set in the rake task?  (I'm thinking a week is too big of a window.)  Let's do "more than X days old" and pass X to the rake task.